src/app/heroes/pages/new-hero-page/new-hero-page.component.html

```diff
@@ -3,7 +3,7 @@ <h2>
   {{ currentHero.superhero }}
 </h2>
 <mat-divider class="mb-2"></mat-divider>
-
+<button mat-flat-button color="primary" (click)="onReturn()">Return</button>
```

Comment:
We can create a new ticket to restyle this view, this botton doesn't look good at all.